examples/kaguya_isis_cmp.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out a USGSCSM ISD from a PDS3 Kaguya image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pvl\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import knoten\n",
+    "import csmapi\n",
+    "\n",
+    "os.environ['ISISROOT'] = '/usgs/pkgs/isis3.8.0_RC1/install'\n",
+    "from pysis import isis\n",
+    "from pysis.exceptions import ProcessError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make a CSM sensor model\n",
+    "Requires TC1S2B0_01_02842S506E1942.img in data directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fileName = 'data/TC1S2B0_01_02842S506E1942.img'\n",
+    "camera = knoten.csm.create_csm(fileName)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the output location of the resulting .cub\n",
+    "cub_loc = 'data/TC1S2B0_01_02842S506E1942.cub'\n",
+    "\n",
+    "try: \n",
+    "    isis.kaguyatc2isis(from_=fileName, to=cub_loc)\n",
+    "except ProcessError as e:\n",
+    "    print(e.stderr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingest the image and spiceinit"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the output location of the resulting .cub\n",
+    "cub_loc = 'data/TC1S2B0_01_02842S506E1942.cub'\n",
+    "\n",
+    "try: \n",
+    "    isis.kaguyatc2isis(from_=fileName, to=cub_loc)\n",
+    "except ProcessError as e:\n",
+    "    print(e.stderr)\n",
+    "\n",
+    "try:\n",
+    "    isis.spiceinit(from_=cub_loc, shape='ellipsoid')\n",
+    "except ProcessError as e:\n",
+    "    print(e.stderr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Define a function that takes an image coordinate, projects it to a ground point using ISIS, then projects the result back into an image coordinate using USGSCSM and computes the difference between image coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def check_pixel(camera, cub, line, sample):\n",
+    "    output = isis.campt(from_=cub, line=line, sample=sample)\n",
+    "    pvl_output = pvl.loads(output)\n",
+    "    bodyfixed = pvl_output['GroundPoint']['BodyFixedCoordinate']\n",
+    "    bodyfixed = np.asarray(bodyfixed.value) * 1000\n",
+    "    image_coord = camera.groundToImage(csmapi.EcefCoord(*bodyfixed))\n",
+    "    line_diff = pvl_output['GroundPoint']['Line'] - image_coord.line - .5\n",
```

Comment:
I might be reading this wrong, but would line and pvl_output['GroundPoint']['Line'] be the same and sample  and pvl_output['GroundPoint']['Sample'] be the same? It is a little confusing to be calculating the difference between something named image_coord and something taken from 'GroundPoint', even though we know this works it might be confusing for someone new to look at. Would it make more sense to subtract line and sample instead?

Reply:
You're totally correct about just using the line and sample parameters passed into the function. We could rename the image_coord variable to something like csm_coord.